src/main/java/seedu/address/commons/util/FileUtil.java

```diff
@@ -80,4 +88,19 @@ public static void writeToFile(Path file, String content) throws IOException {
         Files.write(file, content.getBytes(CHARSET));
     }
 
+    public static String getCurrentPath() throws URISyntaxException {
+        return String.valueOf(Main.class.getProtectionDomain().getCodeSource().getLocation().toURI().getPath());
+    }
+
+    /**
+     * Returns true if a given string is a valid file name.
+     */
+    public static boolean isValidJsonFileName(String test) {
+        if (test.isEmpty()) {
+            return true;
```

Comment:
Just to clarify, should this return False? Should a empty string be considered a valid json file name

Reply:
Oh yeah right thanks for pointing it out